main/main.cpp

```diff
@@ -23,9 +23,8 @@
 #include <Battery.h>
 #include <FuelGauge.h>
 #include <Storage.h>
```

Comment:
```suggestion
#include "Battery.h"
#include "FuelGauge.h"
#include "Storage.h"
```

Reply:
These changes are for other PR cleaning up all the include style. This PR doesn't touches that code though.